src/CMakeLists.txt

```diff
@@ -30,7 +30,7 @@ MACRO(IPOPT_PLUGIN NAME)
   INSTALL(TARGETS roboptim-core-plugin-${NAME}
     DESTINATION ${PLUGINDIR})
   PKG_CONFIG_USE_DEPENDENCY(roboptim-core-plugin-${NAME} ipopt)
-  PKG_CONFIG_USE_COMPILE_DEPENDENCY(roboptim-core-plugin-${NAME} roboptim-core)
+  PKG_CONFIG_USE_DEPENDENCY(roboptim-core-plugin-${NAME} roboptim-core)
```

Comment:
Without the link, there is a lot of missing symbols :

<pre><code>
ipopt.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) class std::basic_ostream<char,struct std::char_traits<char> > & __cdecl roboptim::incindent(class std::basic_ostream<char,struct std::char_traits<char> > &)" (__imp_?incindent@roboptim@@YAAEAV?$basic_ostream@DU?$char_traits@D@std@@@std@@AEAV23@@Z) referenced in function "public: void __cdecl roboptim::detail::printConstraint<class roboptim::Problem<struct roboptim::EigenMatrixDense> >::operator()<class boost::shared_ptr<class roboptim::GenericFunction<struct roboptim::EigenMatrixDense> > >(class boost::shared_ptr<class roboptim::GenericFunction<struct roboptim::EigenMatrixDense> > const &)" (??$?RV?$shared_ptr@V?$GenericFunction@UEigenMatrixDense@roboptim@@@roboptim@@@boost@@@?$printConstraint@V?$Problem@UEigenMatrixDense@roboptim@@@roboptim@@@detail@roboptim@@QEAAXAEBV?$shared_ptr@V?$GenericFunction@UEigenMatrixDense@roboptim@@@roboptim@@@boost@@@Z)

tnlp.cc.obj : error LNK2001: unresolved external symbol "__declspec(dllimport) class std::basic_ostream<char,struct std::char_traits<char> > & __cdecl roboptim::incindent(class std::basic_ostream<char,struct std::char_traits<char> > &)" (__imp_?incindent@roboptim@@YAAEAV?$basic_ostream@DU?$char_traits@D@std@@@std@@AEAV23@@Z)

ipopt.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) class std::basic_ostream<char,struct std::char_traits<char> > & __cdecl roboptim::decindent(class std::basic_ostream<char,struct std::char_traits<char> > &)" (__imp_?decindent@roboptim@@YAAEAV?$basic_ostream@DU?$char_traits@D@std@@@std@@AEAV23@@Z) referenced in function "public: void __cdecl roboptim::detail::printConstraint<class roboptim::Problem<struct roboptim::EigenMatrixDense> >::operator()<class boost::shared_ptr<class roboptim::GenericFunction<struct roboptim::EigenMatrixDense> > >(class boost::shared_ptr<class roboptim::GenericFunction<struct roboptim::EigenMatrixDense> > const &)" (??$?RV?$shared_ptr@V?$GenericFunction@UEigenMatrixDense@roboptim@@@roboptim@@@boost@@@?$printConstraint@V?$Problem@UEigenMatrixDense@roboptim@@@roboptim@@@detail@roboptim@@QEAAXAEBV?$shared_ptr@V?$GenericFunction@UEigenMatrixDense@roboptim@@@roboptim@@@boost@@@Z)

tnlp.cc.obj : error LNK2001: unresolved external symbol "__declspec(dllimport) class std::basic_ostream<char,struct std::char_traits<char> > & __cdecl roboptim::decindent(class std::basic_ostream<char,struct std::char_traits<char> > &)" (__imp_?decindent@roboptim@@YAAEAV?$basic_ostream@DU?$char_traits@D@std@@@std@@AEAV23@@Z)

ipopt.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) class std::basic_ostream<char,struct std::char_traits<char> > & __cdecl roboptim::iendl(class std::basic_ostream<char,struct std::char_traits<char> > &)" (__imp_?iendl@roboptim@@YAAEAV?$basic_ostream@DU?$char_traits@D@std@@@std@@AEAV23@@Z) referenced in function "public: void __cdecl roboptim::detail::printConstraint<class roboptim::Problem<struct roboptim::EigenMatrixDense> >::operator()<class boost::shared_ptr<class roboptim::GenericFunction<struct roboptim::EigenMatrixDense> > >(class boost::shared_ptr<class roboptim::GenericFunction<struct roboptim::EigenMatrixDense> > const &)" (??$?RV?$shared_ptr@V?$GenericFunction@UEigenMatrixDense@roboptim@@@roboptim@@@boost@@@?$printConstraint@V?$Problem@UEigenMatrixDense@roboptim@@@roboptim@@@detail@roboptim@@QEAAXAEBV?$shared_ptr@V?$GenericFunction@UEigenMatrixDense@roboptim@@@roboptim@@@boost@@@Z)

tnlp.cc.obj : error LNK2001: unresolved external symbol "__declspec(dllimport) class std::basic_ostream<char,struct std::char_traits<char> > & __cdecl roboptim::iendl(class std::basic_ostream<char,struct std::char_traits<char> > &)" (__imp_?iendl@roboptim@@YAAEAV?$basic_ostream@DU?$char_traits@D@std@@@std@@AEAV23@@Z)

ipopt.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) class std::basic_ostream<char,struct std::char_traits<char> > & __cdecl roboptim::incendl(class std::basic_ostream<char,struct std::char_traits<char> > &)" (__imp_?incendl@roboptim@@YAAEAV?$basic_ostream@DU?$char_traits@D@std@@@std@@AEAV23@@Z) referenced in function "public: class std::basic_ostream<char,struct std::char_traits<char> > & __cdecl roboptim::Problem<struct roboptim::EigenMatrixDense>::print(class std::basic_ostream<char,struct std::char_traits<char> > &)const " (?print@?$Problem@UEigenMatrixDense@roboptim@@@roboptim@@QEBAAEAV?$basic_ostream@DU?$char_traits@D@std@@@std@@AEAV34@@Z)

tnlp.cc.obj : error LNK2001: unresolved external symbol "__declspec(dllimport) class std::basic_ostream<char,struct std::char_traits<char> > & __cdecl roboptim::incendl(class std::basic_ostream<char,struct std::char_traits<char> > &)" (__imp_?incendl@roboptim@@YAAEAV?$basic_ostream@DU?$char_traits@D@std@@@std@@AEAV23@@Z)

ipopt.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const __cdecl roboptim::demangle(char const *)" (__imp_?demangle@roboptim@@YA?BV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@PEBD@Z) referenced in function "class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > __cdecl roboptim::typeString<class roboptim::GenericDifferentiableFunction<struct roboptim::EigenMatrixDense> >(void)" (??$typeString@V?$GenericDifferentiableFunction@UEigenMatrixDense@roboptim@@@roboptim@@@roboptim@@YA?AV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@XZ)

ipopt.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: __cdecl roboptim::SolverWarning::SolverWarning(class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &)" (__imp_??0SolverWarning@roboptim@@QEAA@AEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@@Z) referenced in function "public: virtual void __cdecl roboptim::detail::Tnlp<class roboptim::IpoptSolver>::finalize_solution(enum Ipopt::SolverReturn,int,double const *,double const *,double const *,int,double const *,double const *,double,class Ipopt::IpoptData const *,class Ipopt::IpoptCalculatedQuantities *)" (?finalize_solution@?$Tnlp@VIpoptSolver@roboptim@@@detail@roboptim@@UEAAXW4SolverReturn@Ipopt@@HPEBN11H11NPEBVIpoptData@5@PEAVIpoptCalculatedQuantities@5@@Z)

ipopt.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: virtual __cdecl roboptim::SolverWarning::~SolverWarning(void)" (__imp_??1SolverWarning@roboptim@@UEAA@XZ) referenced in function "public: virtual void * __cdecl roboptim::SolverWarning::`scalar deleting destructor'(unsigned int)" (??_GSolverWarning@roboptim@@UEAAPEAXI@Z)

ipopt.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: __cdecl roboptim::SolverWarning::SolverWarning(class roboptim::SolverWarning &&)" (__imp_??0SolverWarning@roboptim@@QEAA@$$QEAV01@@Z) referenced in function "class roboptim::SolverWarning * __cdecl std::_Uninit_move<class roboptim::SolverWarning *,class roboptim::SolverWarning *,class std::allocator<class roboptim::SolverWarning>,class roboptim::SolverWarning>(class roboptim::SolverWarning *,class roboptim::SolverWarning *,class roboptim::SolverWarning *,struct std::_Wrap_alloc<class std::allocator<class roboptim::SolverWarning> > &,class roboptim::SolverWarning *,struct std::_Nonscalar_ptr_iterator_tag)" (??$_Uninit_move@PEAVSolverWarning@roboptim@@PEAV12@V?$allocator@VSolverWarning@roboptim@@@std@@V12@@std@@YAPEAVSolverWarning@roboptim@@PEAV12@00AEAU?$_Wrap_alloc@V?$allocator@VSolverWarning@roboptim@@@std@@@0@0U_Nonscalar_ptr_iterator_tag@0@@Z)

ipopt.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: __cdecl roboptim::Result::Result(__int64,__int64)" (__imp_??0Result@roboptim@@QEAA@_J0@Z) referenced in function "public: virtual void __cdecl roboptim::detail::Tnlp<class roboptim::IpoptSolver>::finalize_solution(enum Ipopt::SolverReturn,int,double const *,double const *,double const *,int,double const *,double const *,double,class Ipopt::IpoptData const *,class Ipopt::IpoptCalculatedQuantities *)" (?finalize_solution@?$Tnlp@VIpoptSolver@roboptim@@@detail@roboptim@@UEAAXW4SolverReturn@Ipopt@@HPEBN11H11NPEBVIpoptData@5@PEAVIpoptCalculatedQuantities@5@@Z)

ipopt.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: virtual __cdecl roboptim::Result::~Result(void)" (__imp_??1Result@roboptim@@UEAA@XZ) referenced in function "public: virtual void * __cdecl roboptim::Result::`scalar deleting destructor'(unsigned int)" (??_GResult@roboptim@@UEAAPEAXI@Z)

ipopt.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: __cdecl roboptim::Result::Result(class roboptim::Result const &)" (__imp_??0Result@roboptim@@QEAA@AEBV01@@Z) referenced in function "public: __cdecl boost::variant<class roboptim::NoSolution,class roboptim::Result,class roboptim::SolverError,class roboptim::ResultWithWarnings,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_>::variant<class roboptim::NoSolution,class roboptim::Result,class roboptim::SolverError,class roboptim::ResultWithWarnings,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_><class roboptim::Result>(class roboptim::Result const &)" (??$?0VResult@roboptim@@@?$variant@VNoSolution@roboptim@@VResult@2@VSolverError@2@VResultWithWarnings@2@Uvoid_@0detail@boost@@U6078@U6078@U6078@U6078@U6078@U6078@U6078@U6078@U6078@U6078@U6078@U6078@U6078@U6078@U6078@@boost@@QEAA@AEBVResult@roboptim@@@Z)

ipopt.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: class roboptim::Result & __cdecl roboptim::Result::operator=(class roboptim::Result const &)" (__imp_??4Result@roboptim@@QEAAAEAV01@AEBV01@@Z) referenced in function "public: bool __cdecl boost::variant<class roboptim::NoSolution,class roboptim::Result,class roboptim::SolverError,class roboptim::ResultWithWarnings,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_>::apply_visitor<class boost::detail::variant::direct_assigner<class roboptim::Result> >(class boost::detail::variant::direct_assigner<class roboptim::Result> &)" (??$apply_visitor@V?$direct_assigner@VResult@roboptim@@@variant@detail@boost@@@?$variant@VNoSolution@roboptim@@VResult@2@VSolverError@2@VResultWithWarnings@2@Uvoid_@0detail@boost@@U6078@U6078@U6078@U6078@U6078@U6078@U6078@U6078@U6078@U6078@U6078@U6078@U6078@U6078@U6078@@boost@@QEAA_NAEAV?$direct_assigner@VResult@roboptim@@@variant@detail@1@@Z)

ipopt.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) class std::basic_ostream<char,struct std::char_traits<char> > & __cdecl roboptim::operator<<(class std::basic_ostream<char,struct std::char_traits<char> > &,class roboptim::Result const &)" (__imp_??6roboptim@@YAAEAV?$basic_ostream@DU?$char_traits@D@std@@@std@@AEAV12@AEBVResult@0@@Z) referenced in function "public: void __cdecl boost::detail::variant::printer<class std::basic_ostream<char,struct std::char_traits<char> > >::operator()<class roboptim::Result>(class roboptim::Result const &)const " (??$?RVResult@roboptim@@@?$printer@V?$basic_ostream@DU?$char_traits@D@std@@@std@@@variant@detail@boost@@QEBAXAEBVResult@roboptim@@@Z)

tnlp.cc.obj : error LNK2001: unresolved external symbol "__declspec(dllimport) class std::basic_ostream<char,struct std::char_traits<char> > & __cdecl roboptim::operator<<(class std::basic_ostream<char,struct std::char_traits<char> > &,class roboptim::Result const &)" (__imp_??6roboptim@@YAAEAV?$basic_ostream@DU?$char_traits@D@std@@@std@@AEAV12@AEBVResult@0@@Z)

ipopt.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: __cdecl roboptim::SolverError::SolverError(class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,class roboptim::Result const &)" (__imp_??0SolverError@roboptim@@QEAA@AEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@AEBVResult@1@@Z) referenced in function "public: virtual void __cdecl roboptim::detail::Tnlp<class roboptim::IpoptSolver>::finalize_solution(enum Ipopt::SolverReturn,int,double const *,double const *,double const *,int,double const *,double const *,double,class Ipopt::IpoptData const *,class Ipopt::IpoptCalculatedQuantities *)" (?finalize_solution@?$Tnlp@VIpoptSolver@roboptim@@@detail@roboptim@@UEAAXW4SolverReturn@Ipopt@@HPEBN11H11NPEBVIpoptData@5@PEAVIpoptCalculatedQuantities@5@@Z)

ipopt.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: __cdecl roboptim::SolverError::SolverError(class roboptim::SolverError const &)" (__imp_??0SolverError@roboptim@@QEAA@AEBV01@@Z) referenced in function "public: __cdecl boost::variant<class roboptim::NoSolution,class roboptim::Result,class roboptim::SolverError,class roboptim::ResultWithWarnings,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_>::variant<class roboptim::NoSolution,class roboptim::Result,class roboptim::SolverError,class roboptim::ResultWithWarnings,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_><class roboptim::SolverError>(class roboptim::SolverError &&,void *,void *)" (??$?0VSolverError@roboptim@@@?$variant@VNoSolution@roboptim@@VResult@2@VSolverError@2@VResultWithWarnings@2@Uvoid_@0detail@boost@@U6078@U6078@U6078@U6078@U6078@U6078@U6078@U6078@U6078@U6078@U6078@U6078@U6078@U6078@U6078@@boost@@QEAA@$$QEAVSolverError@roboptim@@PEAX1@Z)

ipopt.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: virtual __cdecl roboptim::SolverError::~SolverError(void)" (__imp_??1SolverError@roboptim@@UEAA@XZ) referenced in function "public: virtual void * __cdecl roboptim::SolverError::`scalar deleting destructor'(unsigned int)" (??_GSolverError@roboptim@@UEAAPEAXI@Z)

ipopt.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: class roboptim::SolverError & __cdecl roboptim::SolverError::operator=(class roboptim::SolverError const &)" (__imp_??4SolverError@roboptim@@QEAAAEAV01@AEBV01@@Z) referenced in function "public: bool __cdecl boost::variant<class roboptim::NoSolution,class roboptim::Result,class roboptim::SolverError,class roboptim::ResultWithWarnings,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_>::apply_visitor<class boost::detail::variant::direct_mover<class roboptim::SolverError> >(class boost::detail::variant::direct_mover<class roboptim::SolverError> &)" (??$apply_visitor@V?$direct_mover@VSolverError@roboptim@@@variant@detail@boost@@@?$variant@VNoSolution@roboptim@@VResult@2@VSolverError@2@VResultWithWarnings@2@Uvoid_@0detail@boost@@U6078@U6078@U6078@U6078@U6078@U6078@U6078@U6078@U6078@U6078@U6078@U6078@U6078@U6078@U6078@@boost@@QEAA_NAEAV?$direct_mover@VSolverError@roboptim@@@variant@detail@1@@Z)

ipopt.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) class std::basic_ostream<char,struct std::char_traits<char> > & __cdecl roboptim::operator<<(class std::basic_ostream<char,struct std::char_traits<char> > &,class roboptim::SolverError const &)" (__imp_??6roboptim@@YAAEAV?$basic_ostream@DU?$char_traits@D@std@@@std@@AEAV12@AEBVSolverError@0@@Z) referenced in function "public: void __cdecl boost::detail::variant::printer<class std::basic_ostream<char,struct std::char_traits<char> > >::operator()<class roboptim::SolverError>(class roboptim::SolverError const &)const " (??$?RVSolverError@roboptim@@@?$printer@V?$basic_ostream@DU?$char_traits@D@std@@@std@@@variant@detail@boost@@QEBAXAEBVSolverError@roboptim@@@Z)

tnlp.cc.obj : error LNK2001: unresolved external symbol "__declspec(dllimport) class std::basic_ostream<char,struct std::char_traits<char> > & __cdecl roboptim::operator<<(class std::basic_ostream<char,struct std::char_traits<char> > &,class roboptim::SolverError const &)" (__imp_??6roboptim@@YAAEAV?$basic_ostream@DU?$char_traits@D@std@@@std@@AEAV12@AEBVSolverError@0@@Z)

ipopt.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: virtual __cdecl roboptim::ResultWithWarnings::~ResultWithWarnings(void)" (__imp_??1ResultWithWarnings@roboptim@@UEAA@XZ) referenced in function "public: virtual void * __cdecl roboptim::ResultWithWarnings::`scalar deleting destructor'(unsigned int)" (??_GResultWithWarnings@roboptim@@UEAAPEAXI@Z)

ipopt.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: __cdecl roboptim::ResultWithWarnings::ResultWithWarnings(class roboptim::ResultWithWarnings const &)" (__imp_??0ResultWithWarnings@roboptim@@QEAA@AEBV01@@Z) referenced in function "private: void __cdecl boost::variant<class roboptim::NoSolution,class roboptim::Result,class roboptim::SolverError,class roboptim::ResultWithWarnings,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_,struct boost::detail::variant::void_>::move_assigner::assign_impl<class roboptim::ResultWithWarnings>(class roboptim::ResultWithWarnings &,struct boost::mpl::bool_<0>,struct boost::mpl::bool_<0>,struct boost::mpl::bool_<1>)const " (??$assign_impl@VResultWithWarnings@roboptim@@@move_assigner@?$variant@VNoSolution@roboptim@@VResult@2@VSolverError@2@VResultWithWarnings@2@Uvoid_@0detail@boost@@U6078@U6078@U6078@U6078@U6078@U6078@U6078@U6078@U6078@U6078@U6078@U6078@U6078@U6078@U6078@@boost@@AEBAXAEAVResultWithWarnings@roboptim@@U?$bool_@$0A@@mpl@2@1U?$bool_@$00@62@@Z)

ipopt.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: class roboptim::ResultWithWarnings & __cdecl roboptim::ResultWithWarnings::operator=(class roboptim::ResultWithWarnings const &)" (__imp_??4ResultWithWarnings@roboptim@@QEAAAEAV01@AEBV01@@Z) referenced in function "public: void __cdecl boost::detail::variant::move_storage::internal_visit<class roboptim::ResultWithWarnings>(class roboptim::ResultWithWarnings &,int)const " (??$internal_visit@VResultWithWarnings@roboptim@@@move_storage@variant@detail@boost@@QEBAXAEAVResultWithWarnings@roboptim@@H@Z)

ipopt.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: __cdecl roboptim::GenericSolver::GenericSolver(void)" (__imp_??0GenericSolver@roboptim@@QEAA@XZ) referenced in function "public: __cdecl roboptim::Solver<struct roboptim::EigenMatrixDense>::Solver<struct roboptim::EigenMatrixDense>(class roboptim::Problem<struct roboptim::EigenMatrixDense> const &)" (??0?$Solver@UEigenMatrixDense@roboptim@@@roboptim@@QEAA@AEBV?$Problem@UEigenMatrixDense@roboptim@@@1@@Z)

tnlp.cc.obj : error LNK2001: unresolved external symbol "__declspec(dllimport) public: __cdecl roboptim::GenericSolver::GenericSolver(void)" (__imp_??0GenericSolver@roboptim@@QEAA@XZ)

ipopt.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: __cdecl roboptim::GenericSolver::GenericSolver(class roboptim::GenericSolver const &)" (__imp_??0GenericSolver@roboptim@@QEAA@AEBV01@@Z) referenced in function "public: __cdecl roboptim::Solver<struct roboptim::EigenMatrixDense>::Solver<struct roboptim::EigenMatrixDense>(class roboptim::Solver<struct roboptim::EigenMatrixDense> const &)" (??0?$Solver@UEigenMatrixDense@roboptim@@@roboptim@@QEAA@AEBV01@@Z)

tnlp.cc.obj : error LNK2001: unresolved external symbol "__declspec(dllimport) public: __cdecl roboptim::GenericSolver::GenericSolver(class roboptim::GenericSolver const &)" (__imp_??0GenericSolver@roboptim@@QEAA@AEBV01@@Z)

ipopt.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: virtual __cdecl roboptim::GenericSolver::~GenericSolver(void)" (__imp_??1GenericSolver@roboptim@@UEAA@XZ) referenced in function "int `public: __cdecl roboptim::Solver<struct roboptim::EigenMatrixDense>::Solver<struct roboptim::EigenMatrixDense>(class Solver<struct roboptim::EigenMatrixDense>::dtor$0 const &)'::`1'::dtor$0" (?dtor$0@?0???0?$Solver@UEigenMatrixDense@roboptim@@@roboptim@@QEAA@AEBV01@@Z@4HA)

tnlp.cc.obj : error LNK2001: unresolved external symbol "__declspec(dllimport) public: virtual __cdecl roboptim::GenericSolver::~GenericSolver(void)" (__imp_??1GenericSolver@roboptim@@UEAA@XZ)

ipopt.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) class std::basic_ostream<char,struct std::char_traits<char> > & __cdecl roboptim::operator<<(class std::basic_ostream<char,struct std::char_traits<char> > &,class roboptim::NoSolution const &)" (__imp_??6roboptim@@YAAEAV?$basic_ostream@DU?$char_traits@D@std@@@std@@AEAV12@AEBVNoSolution@0@@Z) referenced in function "public: void __cdecl boost::detail::variant::printer<class std::basic_ostream<char,struct std::char_traits<char> > >::operator()<class roboptim::NoSolution>(class roboptim::NoSolution const &)const " (??$?RVNoSolution@roboptim@@@?$printer@V?$basic_ostream@DU?$char_traits@D@std@@@std@@@variant@detail@boost@@QEBAXAEBVNoSolution@roboptim@@@Z)

tnlp.cc.obj : error LNK2001: unresolved external symbol "__declspec(dllimport) class std::basic_ostream<char,struct std::char_traits<char> > & __cdecl roboptim::operator<<(class std::basic_ostream<char,struct std::char_traits<char> > &,class roboptim::NoSolution const &)" (__imp_??6roboptim@@YAAEAV?$basic_ostream@DU?$char_traits@D@std@@@std@@AEAV12@AEBVNoSolution@0@@Z)

ipopt.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: __cdecl roboptim::Parameter::Parameter(void)" (__imp_??0Parameter@roboptim@@QEAA@XZ) referenced in function "public: __cdecl std::pair<class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const ,struct roboptim::Parameter>::pair<class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const ,struct roboptim::Parameter><class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > &&>(struct std::piecewise_construct_t,class std::tuple<class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > &&>,class std::tuple<>)" (??$?0$$QEAV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@$$Z$$V@?$pair@$$CBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@UParameter@roboptim@@@std@@QEAA@Upiecewise_construct_t@1@V?$tuple@$$QEAV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@@1@V?$tuple@$$V@1@@Z)

ipopt.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) class std::basic_ostream<char,struct std::char_traits<char> > & __cdecl roboptim::operator<<(class std::basic_ostream<char,struct std::char_traits<char> > &,struct roboptim::Parameter const &)" (__imp_??6roboptim@@YAAEAV?$basic_ostream@DU?$char_traits@D@std@@@std@@AEAV12@AEBUParameter@0@@Z) referenced in function "public: virtual class std::basic_ostream<char,struct std::char_traits<char> > & __cdecl roboptim::Solver<struct roboptim::EigenMatrixDense>::print(class std::basic_ostream<char,struct std::char_traits<char> > &)const " (?print@?$Solver@UEigenMatrixDense@roboptim@@@roboptim@@UEBAAEAV?$basic_ostream@DU?$char_traits@D@std@@@std@@AEAV34@@Z)

tnlp.cc.obj : error LNK2001: unresolved external symbol "__declspec(dllimport) class std::basic_ostream<char,struct std::char_traits<char> > & __cdecl roboptim::operator<<(class std::basic_ostream<char,struct std::char_traits<char> > &,struct roboptim::Parameter const &)" (__imp_??6roboptim@@YAAEAV?$basic_ostream@DU?$char_traits@D@std@@@std@@AEAV12@AEBUParameter@0@@Z)

ipopt.cc.obj : error LNK2001: unresolved external symbol "public: virtual class std::basic_ostream<char,struct std::char_traits<char> > & __cdecl roboptim::Result::print(class std::basic_ostream<char,struct std::char_traits<char> > &)const " (?print@Result@roboptim@@UEBAAEAV?$basic_ostream@DU?$char_traits@D@std@@@std@@AEAV34@@Z)

ipopt.cc.obj : error LNK2001: unresolved external symbol "public: virtual class std::basic_ostream<char,struct std::char_traits<char> > & __cdecl roboptim::SolverError::print(class std::basic_ostream<char,struct std::char_traits<char> > &)const " (?print@SolverError@roboptim@@UEBAAEAV?$basic_ostream@DU?$char_traits@D@std@@@std@@AEAV34@@Z)

ipopt.cc.obj : error LNK2001: unresolved external symbol "public: virtual class std::basic_ostream<char,struct std::char_traits<char> > & __cdecl roboptim::SolverWarning::print(class std::basic_ostream<char,struct std::char_traits<char> > &)const " (?print@SolverWarning@roboptim@@UEBAAEAV?$basic_ostream@DU?$char_traits@D@std@@@std@@AEAV34@@Z)

roboptim-core-plugin-ipopt.dll : fatal error LNK1120: 30 unresolved externals
</code></pre>